src/custom/defines_pre.h

```diff
@@ -7,6 +7,9 @@
  * rAthena configuration file (http://rathena.org)
  * For detailed guidance on these check http://rathena.org/wiki/SRC/config/
  **/
+#define PACKET_OBFUSCATION_KEY1 0x631c511c
+#define PACKET_OBFUSCATION_KEY2 0x111c111c
+#define PACKET_OBFUSCATION_KEY3 0x111c111c
```

Comment:
this should be reverted for the final PR to be merged.

nothing which should be added to the rathena repo.